app/views/docs/security.phtml

```diff
@@ -4,7 +4,7 @@
 
 <h2>SSL Certificates</h2>
 
-<p>Appwrite uses <a href="https://github.com/traefik/traefik" target="_blank" rel="noopener">Traefik</a> and <a href="https://letsencrypt.org/" target="_blank" rel="noopener">Let'sencrypt</a> to serve, create, and renew SSL certificates for your Appwrite server and any custom domains set for your projects.</p>
+<p>Appwrite uses <a href="https://github.com/traefik/traefik" target="_blank" rel="noopener">Traefik</a> and <a href="https://letsencrypt.org/" target="_blank" rel="noopener">Let's Encrypt</a> to serve, create, and renew SSL certificates for your Appwrite server and any custom domains set for your projects.</p>
```

Comment:
```suggestion
<p>Appwrite uses <a href="https://github.com/traefik/traefik" target="_blank" rel="noopener">Traefik</a> and <a href="https://letsencrypt.org/" target="_blank" rel="noopener">Let'sencrypt</a> to serve, create, and renew SSL certificates for your Appwrite server and any custom domains set for your projects.</p>
```

This is a name, it should be a single word.

Reply:
@eldadfux According to their official website, they designate themselves as `Let's Encrypt`, not `Let'sencrypt`:
<img width="1148" alt="Screen Shot 2021-11-06 at 10 50 38 AM" src="https://user-images.githubusercontent.com/33012425/140619097-c370458d-2db0-4ca6-be12-dc00400fb0b2.png">

Reply:
@eldadfux On their official website, they designate themselves as `Let's Encrypt`, not `Let'sencrypt`:
<img width="1148" alt="Screen Shot 2021-11-06 at 10 50 38 AM" src="https://user-images.githubusercontent.com/33012425/140619097-c370458d-2db0-4ca6-be12-dc00400fb0b2.png">